resources/META-INF/plugin-release-info.xml

```diff
@@ -130,6 +130,11 @@
                         <li>Attribute is not defined</li>
                     </ul>
                 </li>
+                <li><i>Feature:</i> Added new inspection rule (<a href="https://github.com/epam/sap-commerce-intellij-idea-plugin/pull/289" target="_blank" rel="nofollow">#289</a>)
```

Comment:
if you would like to group it would be preferable to state that group as follow:
```
<li><i>Feature:</i> New Inspection Rules for <code>items.xml</code>
<ul>
<lI>rule1 (link)</li>
<lI>rule2 (link)</li>
</ul>
</li>
```